<launch>
        <node name="viz_lane" pkg="lane_detection" type="viz_lane" output="screen">
                <rosparam file="$(find lane_detection)/lane_params.yaml" command="load" />
        </node>
</launch>

